library/Freckle/App/Test.hs

```diff
@@ -106,12 +108,12 @@ instance PrimMonad (AppExample app) where
   type PrimState (AppExample app) = PrimState IO
   primitive = liftIO . primitive
 
-instance HasLogger app => Example (AppExample app a) where
+instance Example (AppExample app a) where
   type Arg (AppExample app a) = app
 
   evaluateExample (AppExample ex) params action =
     evaluateExample
-      (action $ \app -> void $ runLoggerLoggingT app $ runReaderT ex app)
+      (action $ \app -> void $ runReaderT ex app)
```

Comment:
And here would fix our test performance regression to, eh?

Reply:
One of them, at least!